src/PAModel/Utility/Utilities.cs

```diff
@@ -453,5 +453,12 @@ public static void VerifyDirectoryExists(ErrorContainer errors, string fullpath)
                 errors.BadParameter($"Directory not found: {fullpath}");
             }
         }
+
+        public static bool IsLocalDatabaseReferencesEmpty(this Entropy entropy)
```

Comment:
This doesn't need to be an extension method, you can just add the method on Entropy.cs

Reply:
Moved the method to Entropy.cs